supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -174,13 +174,15 @@ The practice of having both modes together is often referred to as "hybrid", "ag
 [discrete]
 [[bios]]
 ==== image:images/yes.png[yes] BIOS (noun)
-*Description*: "BIOS" is an abbreviation for basic input and output system. The plural form is "BIOSes".
+*Description*: "BIOS" is an abbreviation for Basic Input/Output System. The plural form is "BIOSes". BIOS is the specific name for the motherboard firmware that provides runtime services for operating systems in older PCs. Modern computers use a different kind of firmware, called either EFI or UEFI. 
 
-*Use it*: yes
+Do not use _BIOS_ as a generic term to refer to computer firmware. Use _firmware_ or a specific phrase such as "UEFI firmware" or "legacy BIOS" instead.
```

Comment:
Hi @sbmetz, t his PR looks good to me but just one question on this bit: Shouldn't BIOS and firmware be in quotations here because it's not within the definition of the terms but rather it's about how to write them (or not in this case). Looking at the contributing guidance that was added recently:

![image](https://user-images.githubusercontent.com/30370363/198079041-4baa4ded-46e1-48b0-adb3-fc23ff98ce80.png)